src/Parser/WindwalkerMonk/Modules/Items/Katsuos.js

```diff
@@ -0,0 +1,34 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class Katsuos extends Module {
```

Comment:
The module's name should be full length (unless it's something like the Priest cloak X'anshi, Shroud of Archbishop Benedictus) - so KatsuosEclipse (remember to rename it in the CombatLogParser too).

Reply:
Alright, done and all references should be updated too.